go/cmd/cli/app/cli.go

```diff
@@ -0,0 +1,118 @@
+package app
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/unkeyed/unkey/go/cmd/cli/commands"
+	"github.com/unkeyed/unkey/go/cmd/cli/commands/deploy"
+)
+
+// CLI represents our command line interface
+type CLI struct {
```

Comment:
how does this manual definition of args, flags and help fit into the bigger picture of our CLI?

How do you envision the other commands to work here?